packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -617,14 +624,14 @@ const PersonalDetailsForm = props => {
 
 export default PersonalDetailsForm;
 
-const PhoneField = ({ value, editable_fields, has_real_account, required }) => (
+const PhoneField = ({ value, editable_fields, is_disabled, required }) => (
```

Comment:
to avoid confusion, do you think the better name would be `is_poi_verified` instead of is_disabled?

Reply:
i thought so because the component is local. ok, we can go with this 👍🏼